ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -82,13 +93,7 @@ const AdvancedGasFeeDefaults = () => {
         isChecked={isDefaultSettingsSelected}
         onChange={handleUpdateDefaultSettings}
         isDisabled={gasErrors.maxFeePerGas || gasErrors.maxPriorityFeePerGas}
-        label={
-          isDefaultSettingsSelected
-            ? t('advancedGasFeeDefaultOptOut')
-            : t('advancedGasFeeDefaultOptIn', [
-                <strong key="default-value-change">{t('newValues')}</strong>,
-              ])
-        }
+        label={t('advancedGasFeeDefaultOptIn', [capitalize(networkIdentifier)])}
```

Comment:
Good catch, this is much nicer now that the same message is used for both states